OCP/getBounds.m

```diff
@@ -160,6 +164,13 @@
 bounds.Qdots.lower(idx_mtp) = -13;
 bounds.Qdotdots.upper(idx_mtp) = 500;
 bounds.Qdotdots.lower(idx_mtp) = -500;
+% Midtarsal
+bounds.Qs.upper(idx_mtj) = 30*pi/180;
```

Comment:
only evaluatie this if idx_mtj is not empty ?

Reply:
This will automatically do nothing is idx_mtj is empty. PR#89 has an updated implementation of how bounds are determined.